modules/javascript/src/main/webapp/javascript/atmosphere.js

```diff
@@ -1972,7 +1979,7 @@
                             _response.error = true;
                             _response.ffTryingReconnect = true;
                             try {
-                                _response.status = XMLHttpRequest.status;
```

Comment:
`XMLHttpRequest` does not have a `status` field. I'm assuming we want here the `status` from `ajaxRequest`, the instance of `XMLHttpRequest`.